marimbabot_planning/README.md

```diff
@@ -0,0 +1,23 @@
+# TAMS Master Project 2022/2023 - motion documentation
+
+In order to run the whole project on the real robot, launch the bringup package that brings up the launch file for each package:
```

Comment:
This should be removed, as it is already mentioned and not part of this package. :) I would say